src/iminuit/util.py

```diff
@@ -947,6 +953,8 @@ def _key2index(var2pos, key):
         start = _key2index(var2pos, sl.start) if sl.start is not None else None
         stop = _key2index(var2pos, sl.stop) if sl.stop is not None else None
         return slice(start, stop, sl.step)
+    if isinstance(key, tuple) or isinstance(key, list):
```

Comment:
Only lists are allowed here to be one the safe side. Tuples have special meaning in getitem/setitem.